Doc/source/datamodel.rst

```diff
@@ -1,45 +1,169 @@
+#########
+Datamodel
+#########
 
+The datamodel classes constitute a data model implementation
+meant to mirror the functionality of the data model output from pycdf, though
+implemented slightly differently.
 
-##########################################
-datamodel - easy to use general data model
-##########################################
-
-.. automodule:: spacepy.datamodel
-
-.. currentmodule:: spacepy.datamodel
-
-.. rubric:: Classes
-
-.. autosummary::
-    :template: clean_class.rst
-    :toctree: autosummary
-
-    SpaceData
-    dmarray
-    ISTPContainer
-    ISTPArray
-    DMWarning
-
-.. rubric:: Functions
-
-.. autosummary::
-    :toctree: autosummary
-
-    convertKeysToStr
-    createISTPattrs
-    dmcopy
-    dmfilled
-    flatten
-    fromCDF
-    fromHDF5
-    fromRecArray
-    toCDF
-    toHDF5
-    toHTML
-    toJSONheadedASCII
-    toRecArray
-    unflatten
-    readJSONMetadata
-    readJSONheadedASCII
-    resample
-    writeJSONMetadata
+.. contents:: Table of Contents
+    :depth: 2
+    :local:
+
+See also the `full API documentation <spacepy.datamodel>`.
+
+Documentation
+=============
+
+This contains the following classes:
+ * :py:class:`dmarray` - numpy arrays that support .attrs for information about the data
+ * :py:class:`SpaceData` - base class that extends dict, to be extended by others
```

Comment:
We almost certainly don't need the py domain specified here.

Reply:
This was resolved